lib/chewy/type/import.rb

```diff
@@ -66,6 +70,7 @@ def bulk options = {}
           suffix = options.delete(:suffix)
           bulk_size = options.delete(:bulk_size)
           body = options.delete(:body)
+          journal = options.delete(:journal)
```

Comment:
Ok, I've just found that delete works as destructive fetch, so you can simply do `journal.add(action_objects) if import_options.delete(:journal) { journal? }` above